defaults/snapshotter_default_linux.go

```diff
@@ -14,7 +14,7 @@
    limitations under the License.
 */
 
-package containerd
+package defaults
```

Comment:
@Iceber Thank you for all of your helpful feedback. I will look into doing this but I do not know if this could work, as there seems to be conflicting build constraints on some of the files (ex. `snapshotter_default_unix.go` has the build constraint `go:build darwin || freebsd || solaris` while `defaults_unix.go` has the build constraint `go:build !windows && !darwin`).

Reply:
This is indeed a problem, `snapshotter_default_unix.go` will not work for **openbsd** or other systems, which means that the client cannot be used for those systems. It is true that there is special support for **freebsd** in the [native snapshotter](https://github.com/containerd/containerd/blob/main/snapshots/native/native_freebsd.go) and [mount](https://github.com/containerd/containerd/blob/main/mount/mount_freebsd.go) packages, but **openbsd** does not work.

If we want to be consistent, we can move DefaultSnapshotter to defaults_unix.go/defaults_drawin.go, and add support to native snapshotter if there is a future need for openbsd, or we can create a new file for snapshotter, as we are doing now.
I'm not sure, @Jenkins-J @samuelkarp what do you think?